irods/test/data_obj_test.py

```diff
@@ -903,9 +903,10 @@ def test_replica_number(self):
             self.assertEqual(replica.number, i)
 
         # now trim odd-numbered replicas
+        options = {kw.COPIES_KW: 1}
```

Comment:
Noting for posterity that `COPIES_KW` has been deprecated and will be removed in a future release: https://github.com/irods/irods/issues/4861

What you're doing here is correct and necessary for trim to work properly, but I just wanted to make it known that this will change in the distant future. Maybe worth a comment or something in the code?